javaclient/src/main/java/confluo/PropertiesParser.java

```diff
@@ -0,0 +1,32 @@
+package confluo;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+public class PropertiesParser {
+
+    /**
+     *  parse unique domain properties
+     *  @return  filtered properties by domain
+     **/
+    public static Properties parse(String domain, String file) throws IOException{
+        URL url= PropertiesParser.class.getClassLoader().getResource(file);
```

Comment:
The code formatting seems to be off (e.g., 2 spaces indentation, spaces before `=`, etc.). We use Google style for code formatting, please see here: https://github.com/google/google-java-format

Reply:
have reformatted my code by google-java-format